resources/web/server/simulation_server.py

```diff
@@ -208,13 +278,13 @@ def runWebotsInThread(client):
                 if line.startswith('open'):  # Webots world is loaded, ready to receive connections
                     break
             hostname = client.client_websocket.request.host.split(':')[0]
-            if config['ssl']:
-                protocol = 'wss:'
-            else:
-                protocol = 'ws:'
+            protocol = 'wss:' if config['ssl'] or config['portRewrite'] else 'ws:'
+            separator = '/' if config['portRewrite'] else ':'
             asyncio.set_event_loop(asyncio.new_event_loop())
-            client.client_websocket.write_message('webots:' + protocol + '//' +
-                                                  hostname + ':' + str(port))
+            message = 'webots:' + protocol + '//' + hostname + separator + str(port)
+            if config['portRewrite']:
+                message += '/'  # so that the URL is valid
```

Comment:
The extra slash is added here.
The message received by `webots.min.js` if `portRewrite` is enabled has the form:
```
webots:wss://localhost/2003/
```

I get the texture URL error on Chrome/Linux.

Reply:
So, if you simply remove the `+ '/'` from: https://github.com/cyberbotics/webots/blob/202778fa3f6c504a3e3d72c0fa8b3bf18a418e69/resources/web/wwi/server.js#L89 that should fix the problem, right?

Reply:
Yes, it works for the `portRewrite` case.
But it doesn't work for the previous normal case where the separator will be missing.

Reply:
OK, it should be fixed now.